.travis.yml

```diff
@@ -44,16 +47,18 @@ matrix:
         # Try Astropy development version
         - python: 2.7
           env: ASTROPY_VERSION=development SETUP_CMD='test'
-        - python: 3.3
+        - python: 3.4
           env: ASTROPY_VERSION=development SETUP_CMD='test'
 
-        # Try all python versions with the latest numpy
+        # Try with optional dependencies disabled
         - python: 2.7
           env: SETUP_CMD='test'
-        - python: 3.3
-          env: SETUP_CMD='test'
+               CONDA_DEPENDENCIES='requests'
```

Comment:
@bsipocz - we don't currently install WCSAxes as a dependency for the core on Travis, right? Do you think we should?